Source/WebDriver/soup/WebSocketServerSoup.cpp

```diff
@@ -162,6 +162,17 @@ std::optional<String> WebSocketServer::listen(const String& host, unsigned port)
     return getWebSocketURL(m_listener, nullString());
 }
 
+void WebSocketServer::sendMessage(const String& session, const String& message)
+{
+    for (const auto& pair : m_connectionToSession) {
+        if (pair.value == session) {
+            GRefPtr<GBytes> rawMessage = adoptGRef(g_bytes_new(message.utf8().data(), message.length()));
```

Comment:
The lengght of the utf16 string might not be the same as the utf8 conversion, no? Why don't we just pass a CString to sendMessage? I guess all ports need the message as utf8

Reply:
Did you miss this comment?